r2r/client/base.py

```diff
@@ -90,6 +90,14 @@ def search(
         response = requests.post(url, json=json_data)
         return response.json()
 
+    def rag_chatbot(self, query: str):
```

Comment:
I'm not sure we need to introduce an additional endpoint here.

I believe that we should be able to use the existing rag_completion endpoint.